### ISSUE TITLE
Change About & Latest Link Text Size

### ISSUE THREAD

**Issue by merichar** [Dezudio/dezudio-website#4]
Change About and Latest link text size to 25px/28 on xl bp (currently 18/24)

**Comment by merichar** [comment]
@h-w-koenig Looks 25/28 to me?
![2021-08-09-155004_764x695_scrot](https://user-images.githubusercontent.com/304241/128765733-f7d84d07-9713-40ce-8d79-fb2534cba41e.png)

**Comment by h-w-koenig** [comment]
@merichar Yes you are right! We must have been looking at the 1920 breakpoint.
